consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,43 @@
-# Create your models here.
 from django.db import models
 
 
+class Consultation(models.Model):
+    # May need something better long-term
+    class Organisation(models.TextChoices):
+        DFE = "dept-for-education", "Department for Education"
+        CO = "cabinet-office", "Cabinet Office"
+
+    identifier = models.CharField(max_length=256, blank=False)
```

Comment:
This is called `slug` in `Question` - we should be consistent

Reply:
Have updated.